docs/registry/README.md

```diff
@@ -330,3 +331,62 @@ spec:
 Note that more policies should probably need to be configured, e.g., allow auto-registration of EventTypes received 
 from within the cluster (e.g., from a response of a Service running in the cluster) as opposed to external services, and so on. 
 We just enumerate three simple cases here.
+
+## FAQ
+
+Here is a list of frequently asked questions that may help clarify the scope of the Registry.
+
+- Is the Registry meant to be used just for creating Triggers or for also setting up Event Sources?
+
+    It's mainly intended for helping the `Event Consumer` with the `discoverability` use case. Therefore, 
+    it is meant for helping out creating Triggers.
+
```

Comment:
Yes, for MVP of the registry, the expectation is that in order for a user to create a Trigger, they will need to know what event types are available, and hence there should be a way to discover them.
For MVP, these are Event Types that have been configured to be sent to a given broker, hence a Trigger for them can be created.

I think Event Source CRD is a "Potential" Event. Without instantiating one, you have no events. Hence the "Potential". Once an Event Source CR has been created, then those events are available to create Triggers for.

Now, I could certainly see a world (not in MVP), where an Event Source CRD could be pointed to by a registry. 
So, Event Source CRD could be added to a registry, roughly looking like this:
 {Broker, Source, Type} -> Ref To the CRD.

This way if a user looks at the Registry, they would see the Event Types available. Now, in order to actually create the CR, they will probably need to specify additional information (Like Credentials, etc.), and this is where things get wonky because you don't want to necessarily put those into the Trigger spec.

So, I think that' goal is doable, I just think by providing the MVP experience that focuses on the discovery of event types available (not potentially available) would allow us to provide value to the user and then add more niceties on top of it later.

Reply:
Furthermore, it's not only the matter of say providing credentials, there needs to be a way to describe the "subscription schema" that a user can inspect and understand things like, what format the credentials are (unless we come up with a standard, convention, or something like that), but in any case, I just think that providing a Registry as a concept makes users lives better, then in the next release we could start tackling these additional trickier bits.